pkg/sdk/testint/functions_integration_test.go

```diff
@@ -0,0 +1,407 @@
+package testint
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_CreateFunctions(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	_, warehouseCleanup := createWarehouse(t, client)
+	t.Cleanup(warehouseCleanup)
+	databaseTest, databaseCleanup := createDatabase(t, client)
+	t.Cleanup(databaseCleanup)
+	schemaTest, schemaCleanup := createSchema(t, client, databaseTest)
+	t.Cleanup(schemaCleanup)
```

Comment:
why are we using new warehouse/database/schema and not predefined ones ?

Reply:
i didnt realize we had helper functions. i have fixed